litellm/utils.py

```diff
@@ -5921,15 +5921,15 @@ def get_supported_openai_params(model: str, custom_llm_provider: str):
         elif model.startswith("anthropic"):
             return litellm.AmazonAnthropicConfig().get_supported_openai_params()
         elif model.startswith("ai21"):
-            return ["max_tokens", "temperature", "top_p", "stream"]
+            return ["max_tokens", "temperature", "top_p", "stream", "extra_headers"]
```

Comment:
@themrzmaster 'extra_headers' isn't an openai param afaik. 


LiteLLM should already be passing this straight through to bedrock, without needing this code block. 

Please can you remove this